### ISSUE TITLE
Build failure in Xamarin.Forms 4

### ISSUE THREAD

**Issue by sander1133** [jsmarcus/Iconize#115]
In Xamarin.Forms 4 i get the build error: Can't resolve the reference 'UIKit.UIBarButtonItem Xamarin.Forms.Platform.iOS.ToolbarItemExtensions::ToUIBarButtonItem(Xamarin.Forms.ToolbarItem,System.Boolean)'.

Seems like something has changed in de the Xamarin Api.

**Comment by GalaxiaGuy** [comment]
`ToUIBarButtonItem` has gained an extra parameter. Both are optional so I'm not sure why it fails but that passing `false` as the second parameter should work.

https://github.com/xamarin/Xamarin.Forms/blob/0a67f8b0a105fbf56b6ac704cfc283840a189817/Xamarin.Forms.Platform.iOS/Extensions/ToolbarItemExtensions.cs#L12

**Comment by geskill** [comment]
Yep, same problem with recent 3.5 release. Guess we just need a recompile and publish @jsmarcus

**Comment by jsmarcus** [comment]
Pushing a 3.5 build to nuget now
